README.md

```diff
@@ -1,61 +1,147 @@
+The YAML fragments below shows some common options for specifying the target database and table name for the selected tables.
 
-    $SHUNTING_YARD_HOME/bin/replicator.sh --config=$SHUNTING_YARD_HOME/conf/<my-config>.yml
-    
-### Graphite Metrics
+#### Specify both target database and table name
+
+    table-replications:
+      - source-table:
+          database-name: source_database
+          table-name: test_table
+        replica-table:
+          database-name: replica_database
+          table-name: test_table_1    
+          
+#### Only Change the target database but the table name remains same as source
+
+    table-replications:
+      - source-table:
+          database-name: source_database
+          table-name: test_table
+        replica-table:
+          database-name: replica_database 
+
+#### Only Change the target table name but the database remains same as source
+
+    table-replications:
+      - source-table:
+          database-name: source_database
+          table-name: test_table
+        replica-table:
+          table-name: test_table_1
+
+### Shunting Yard configuration reference
+The table below describes all the available configuration values for Shunting Yard.
+
+|Property|Required|Description|
+|:----|:----:|:----|
+|`source-catalog.name`|Yes|A name for the source catalog for events and logging.|
+|`source-catalog.hive-metastore-uris`|No|Fully qualified URI of the source cluster's Hive metastore Thrift service.|
+|`replica-catalog.name`|Yes|A name for the replica catalog for events and logging.|
+|`replica-catalog.hive-metastore-uris`|Yes|Fully qualified URI of the replica cluster's Hive metastore Thrift service.|
+|`event-receiver.configuration-properties.com.hotels.shunting.yard.event.receiver.sqs.queue`|Yes|Fully qualified URI of the [AWS SQS](https://aws.amazon.com/sqs/) Queue to read the hive events from.|
+|`event-receiver.configuration-properties.com.hotels.shunting.yard.event.receiver.sqs.wait.time.seconds`|No|Wait time in seconds for which the receiver will poll the SQS queue for a batch of messages. Default is 10 seconds. Read more about long polling with AWS SQS [here](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-long-polling.html)|
+|`source-table-filter.table-names`|No|A list of tables selected for Shunting Yard replication. Supported Format:`database_1.table_1, database_2.table_2`|
+|`table-replications[n].source-table.database-name`|No|The name of the database in which the table you wish to replicate is located.|
+|`table-replications[n].source-table.table-name`|No|The name of the table which you wish to replicate.|
+|`table-replications[n].replica-table.database-name`|No|The name of the destination database in which to replicate the table. Defaults to source database name.|
+|`table-replications[n].replica-table.table-name`|No|The name of the table at the destination. Defaults to source table name.|
+
+### Configuring Graphite Metrics
```

Comment:
I think "Metrics" should be with lowercase m. "System architecture" has 2 hashes (compared to 3 here) and doesn't have uppercase for each letter in the subtitle, so this shouldn't either.

Reply:
done.

Reply:
Graphite can stay with uppercase, because it's the name of the tool, right?

Reply:
Yeah